src/app/modules/time-clock/pages/time-clock.component.ts

```diff
@@ -1,3 +1,8 @@
+import { LoadActivities } from '../../activities-management/store/activity-management.actions';
+import { ActivityState } from '../../activities-management/store/activity-management.reducers';
+import { Store, select } from '@ngrx/store';
+import { allActivities } from '../../activities-management/store';
+import { Activity } from 'src/app/modules/shared/models';
```

Comment:
Please, organize imports by module.
**import { Component, OnInit } from '@angular/core';**
**import { Store, select } from '@ngrx/store';**
**the rest of imports..**

And if there is an index file in the store directory, you can import in one line

Reply:
done